src/python/pants/backend/jvm/tasks/coursier/coursier_subsystem.py

```diff
@@ -36,7 +36,7 @@ def register_options(cls, register):
     register('--fetch-options', type=list, fingerprint=True,
              help='Additional options to pass to coursier fetch. See `coursier fetch --help`')
     register('--bootstrap-jar-url', fingerprint=True,
-             default='https://dl.dropboxusercontent.com/s/iqdg7iht09rlvsy/coursier-cli-1.0.0.d5de1477516e3429f4d0a522fc0361f2d7b27944.jar?dl=0',
+             default='https://dl.dropboxusercontent.com/s/nd3lfi6fqvlyn9q/coursier-cli.jar?dl=0',
```

Comment:
Could you rename the `coursier-cli.jar` to include the version and sha. I think it's 1.0.2 now, so `coursier-cli-1.0.2.<the sha you built the jar with>.jar`, so it would be more clear where the jar is coming from?